modules/openapi-generator/src/main/resources/JavaJaxRS/cxf-cdi/pojo.mustache

```diff
@@ -33,12 +38,37 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {{#seriali
   {{#vendorExtensions.extraAnnotation}}{{{vendorExtensions.extraAnnotation}}}{{/vendorExtensions.extraAnnotation}}
   @ApiModelProperty({{#example}}example = "{{{example}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}value = "{{{description}}}")
   @JsonProperty("{{baseName}}")
-{{#useBeanValidation}}{{>beanValidation}}{{/useBeanValidation}}  public {{{datatypeWithEnum}}} {{getter}}() {
+{{#useBeanValidation}}{{>beanValidation}}{{/useBeanValidation}}  public {{{datatypeWithEnum}}} {{#isBoolean}}is{{/isBoolean}}{{getter}}() {
```

Comment:
Are you sure about the `{{#isBoolean}}is{{/isBoolean}}{{getter}}` syntax? because I have removed it in OpenAPI Generator.

Reply:
oh, right, thanks !
It looks like I pulled this change from the original one I did for swagger-codegen by error. Correcting it just now.